src/rbprm-rom-validation.cc

```diff
@@ -100,11 +101,11 @@ namespace hpp {
 
     void RbPrmRomValidation::randomnizeCollisionPairs(){
       std::vector<CollisionPair_t> v;
-      v.reserve(collisionPairs_.size());
-      v.insert(v.end(),collisionPairs_.begin(),collisionPairs_.end());
+      v.reserve(pairs().size());
+      v.insert(v.end(),pairs().begin(),pairs().end());
       std::random_shuffle(v.begin(), v.end());
-      collisionPairs_.clear();
-      collisionPairs_.insert(collisionPairs_.end(),v.begin(),v.end());
+      pairs().clear();
+      pairs().insert(pairs().end(),v.begin(),v.end());
```

Comment:
I would change this code so that the requests are also reordered. For instance :

      for (std::size_t i = 0; i < cPairs_.size(); ++i)
      {
        // XXX rand() % N is not uniformly distributed
        std::size_t j = std::rand() % (i + 1);
        if (i != j) {
          std::swap(cPairs_   [i], cPairs_   [j]);
          std::swap(cRequests_[i], cRequests_[j]);
        }
      }

Reply:
Done in eab7b4a.